up.sh

```diff
@@ -77,18 +77,17 @@ function copy_ips_to_s3() {
         "$WORKING_DIR/$testnet_channel-genesis-ip" \
         "s3://safe-testnet-tool/$testnet_channel-genesis-ip" \
         --acl public-read
-}
-
-function update_local_state() {
-    ./scripts/register_keys.sh
-    ./scripts/get-connection-infos
-    local testnet_channel=$(terraform workspace show)
     aws s3 cp \
         "$WORKING_DIR/$testnet_channel-node_connection_info.config" \
         "s3://safe-testnet-tool/$testnet_channel-node_connection_info.config" \
         --acl public-read
 }
 
+function update_local_state() {
+    ./scripts/register_keys.sh
+}
```

Comment:
Can be skipped since we do `ssh-keyscan -H ...` individually as the nodes fire up.